print-sll.c

```diff
@@ -408,6 +415,10 @@ sll2_if_print(netdissect_options *ndo, const struct pcap_pkthdr *h, const u_char
 	}
 
 	sllp = (const struct sll2_header *)p;
+#ifdef HAVE_NET_IF_H
+	if (ndo->ndo_live_capture && if_indextoname(EXTRACT_BE_U_4(sllp->sll2_if_index), ifname))
+		ND_PRINT("IFNAME %s ", ifname);
+#endif
 
```

Comment:
Most of the time the call will be successful. But when the packet reaches tcpdump after the interface was deleted (which is more likely with low packet rate and TPACKET_V3, which can buffer packets long enough), the ifindex at this point will refer to a non-existent interface:
>       On success, if_indextoname() returns ifname; on error, NULL is returned
>       and errno is set appropriately.

Reply:
Maybe "ifindex %u (%s)" on success and just "ifindex %u" on failure?

Reply:
Changed it to suggested "IFNAME %s (%u) " on success on live capture otherwise "IFNAME %u ". 

Do we want to show "IFNAME %s (%u) " also while reading packet? Not sure about it (on the same localhost it's a benefit, but can be misleading).

Reply:
When I'm thinking about it, printing always interface name looks better for me. I'll remove `ndo_live_capture` and give a warning when not live capture.